firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -373,6 +379,29 @@ private boolean isAuthTokenExpired(PersistedFidEntry persistedFidEntry) {
   private long currentTimeInSecs() {
     return TimeUnit.MILLISECONDS.toSeconds(clock.currentTimeMillis());
   }
+
+  interface CombinedListener extends OnCompleteListener<Void> {}
+
+  private static final class AwaitListener implements CombinedListener {
+    private final CountDownLatch mLatch = new CountDownLatch(1);
+
+    public void onSuccess() {
+      mLatch.countDown();
+    }
+
```

Comment:
imo don't bother including this (the argumentless await() version), we shouldn't ever block without a timeout

Reply:
Done. Thanks.

Reply:
Done. Thanks.